client/chat/action-creators.js

```diff
@@ -54,7 +55,10 @@ export function sendMessage(channel, message) {
     })
     dispatch({
       type: CHAT_SEND_MESSAGE,
-      payload: siteSocket.invoke('/chat/send', params),
+      payload: fetch(apiUrl`chat/${channel}`, {
```

Comment:
Seeing this in use here, maybe this would make more sense as a separate sub-url (e.g. `/chat/<channel>/messages`)? This feels like it'll be a bit weird when we let people change settings of the channel, which would likely be the same URL and method